_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -70,12 +89,54 @@ proc _cmake_get_build_dir {} {
     return [option worksrcpath]
 }
 
-default configure.dir {[_cmake_get_build_dir]}
-
-pre-configure {
-    file mkdir ${configure.dir}
+option_proc cmake.generator cmake::handle_generator
+proc cmake::handle_generator {option action args} {
+    global cmake.generator destroot destroot.target build.cmd build.post_args
+    global depends_build destroot.post_args build.jobs
+    if {${action} eq "set"} {
+        switch -glob ${args} {
```

Comment:
I think there's a misunderstanding about doing this change after a squash (which I didn't yet do), I wasn't very enthusiastic about this change at all.
I'll have another look, but tomorrow at the earliest.